### DISCUSSION TITLE
eBusD via TASMOTA

### DISCUSSION THREAD

**Discussion by leonundjulie** [john30/ebusd#760] [Q&A]
Hi to the community. I want to connect my WOLF GASTHERME CGB-K 20 via eBus to my smart home system. I wonder, if there is a way to use the **ebusd** via an ESP8266, flashed with **TASMOTA**.  

Is this possible?
Does someone has some experience?

**Comment by kofec** [comment]
Does any one know/try if is it possible to use Serial-to-TCP-Bridge implemented in Tasmota ?
If so what should we set for TCPBaudRate :
TCPBaudRate : sets the baud rate for serial (only 8N1 mode), min 1200, max 115200 by 1200 increments.
https://tasmota.github.io/docs/Serial-to-TCP-Bridge/
As I understand it should be OK: set GPIO1 as TCP Tx and GPIO3 as TCP Rx.
And port set to: 9999
